assets/sass/atoms/_dropdown.scss

```diff
@@ -0,0 +1,107 @@
+@charset 'UTF-8';
+
+.dropdown {
```

Comment:
Are you sure the Bootstrap dropdown is used nowhere else for other purposes? Your changes here would be repercuted.